### PR TITLE
docs: create a sequence diagram for the `add` command.

### PR DESCRIPTION
**Pull request by Zer0Legion**
Now a more detailed sequence diagram is available for readers to understand the inner workings of a non-trivial command.

Lets strive to improve documentation.

### CODE REVIEW

**Review by Pughal77 — COMMENTED** [review]
This addition is a ok for me!

**Review by delishad21 — COMMENTED** [review]
Very comprehensive diagram! Good job

**Review by jayllo-c — COMMENTED** [review]
nice job on the addition of the sequence diagram

File: docs/diagrams/AddSequenceDiagram.puml
[the remark this answers is not in the document]
Comment (jayllo-c) [reply docs/diagrams/AddSequenceDiagram.puml:1]: After further analysis, there does seem to be an issue with the activation bar for the addCommand object.

**Comment by danielleloh** [comment]
everything looks good to me :))

**Pull request merged**